Dialogs/ProfessionDialog.hpp

```diff
@@ -2,26 +2,17 @@ class IntProfessionDialog
 {
     idd = 2999;
     movingenable = false;
-    
-    class Objects 
-    {
-		class RscModel_2230 : RscObject
-		{
-			idc = 2230;
-			scale = 0.5;
-			z = 8.5;
-			//model = "\A3\armor_f_beta\APC_Tracked_01\APC_Tracked_01_rcws_F.p3d";
-			x = 13 * GUI_GRID_W + GUI_GRID_X;
-			y = 6 * GUI_GRID_H + GUI_GRID_Y;
-			w = 27.5 * GUI_GRID_W;
-			h = 10 * GUI_GRID_H;
-		};
-		
-	};
-
 
 	class Controls
     {
+	    class RscModel_2231 : RscPicture
```

Comment:
What about VehiclePicture? (Or IntProfessionDialog_VehiclePicture) - if that's what it is...